src/bots/pythCranker.ts

```diff
@@ -465,10 +473,17 @@ export class PythCrankerBot implements Bot {
 							doSimulation: true,
 							recentBlockhash: await this.getBlockhashForTx(),
 						});
+
 						const startTime = Date.now();
+						const sendSlot = this.slotSubscriber.getSlot();
+
 						this.driftClient
 							.sendTransaction(simResult.tx)
 							.then((txSigAndSlot: TxSigAndSlot) => {
+								this.txRecorder.send(
+									Date.now() - startTime,
+									txSigAndSlot.slot - sendSlot
+								);
```

Comment:
can you add the `txSigAndSlot.slot` to the log msg below too? im curious whether that's actually the landed slot (hope it is)